web/src/locales/en.json

```diff
@@ -296,7 +299,8 @@
       "enable-password-login-warning": "This will enable password login for all users. Continue only if you want to users to be able to log in using both SSO and password",
       "max-upload-size": "Maximum upload size (MiB)",
       "max-upload-size-hint": "Recommended value is 32 MiB.",
-      "server-name": "Server Name"
+      "server-name": "Server Name",
+      "enable-remove-done": "Enable remove done"
```

Comment:
Can this translation be more descriptive
e.g. `Enable removal of completed checklist items`

Reply:
Okay, I'll change the description